apps/vercel/frontend/src/clients/Vercel.ts

```diff
@@ -24,13 +28,13 @@ export default class VercelClient implements VercelAPIClient {
     });
   }
 
-  async checkToken(): Promise<boolean> {
+  async checkToken(): Promise<CheckTokenResponse> {
     const res = await fetch(`${this.baseEndpoint}/v5/user/tokens`, {
       headers: this.buildHeaders(),
       method: 'GET',
     });
 
-    return res.ok;
+    return { ok: res.ok };
```

Comment:
I adjusted the return of this to ensure that we can determine when the call is finished. This was difficult to do when the return of the call was boolean itself.